samples/client.c

```diff
@@ -690,40 +758,40 @@ get_dirty_bitmap(int sock, struct vfio_user_dma_map *dma_region)
 {
     uint64_t bitmap_size = _get_bitmap_size(dma_region->size,
                                             sysconf(_SC_PAGESIZE));
-    struct vfio_user_dirty_pages *dirty_pages;
-    struct vfio_user_bitmap_range *range;
+
+    struct vfio_user_device_feature *res;
+    struct vfio_user_device_feature_dma_logging_report *report;
+    
     char *bitmap;
-    size_t size;
-    void *data;
     int ret;
 
-    size = sizeof(*dirty_pages) + sizeof(*range) + bitmap_size;
+    ssize_t size = sizeof(*res) + sizeof(*report) + bitmap_size;
```

Comment:
Why change to signed?

Reply:
I have no idea, must've been a mistake! I'll change it back.